kofta/src/vscode-webview/components/ProfileButton.tsx

```diff
@@ -16,9 +16,13 @@ export const ProfileButton: React.FC<ProfileButtonProps> = ({
 }) => {
   const [me] = useAtom(meAtom);
   const history = useHistory();
+  const [open, toggleOpen] = useRoomChatStore((s) => [s.open, s.toggleOpen]);
   return me ? (
     <button
-      onClick={() => history.push("/me")}
+      onClick={() => {
+        if (open) toggleOpen();
+        history.push("/me");
+      }}
```

Comment:
and here?

Reply:
To close chat when entering the user profile